mlir/include/mlir/Dialect/Tosa/IR/TosaOpBase.td

```diff
@@ -381,6 +382,34 @@ class Extension<list<I32EnumAttrCase> extensions> : Availability {
   let instance = "ref";
 }
 
+//===----------------------------------------------------------------------===//
+// Iterable attributes.
+//===----------------------------------------------------------------------===//
+// Defined in `section 3. Enumerations` of the TOSA specification.
+
+def Tosa_RESIZE_BILINEAR          : I32EnumAttrCase<"BILINEAR", 1>;
```

Comment:
It would be preferable to align the numeric values with the canonical ordering defined in the TOSA Spec:
https://mltech.ml.arm.com/spec/tosa/nightly-stable/tosa_spec.html#_enumerations

```
def Tosa_RESIZE_NEAREST_NEIGHBOR  : I32EnumAttrCase<"NEAREST_NEIGHBOR", 1>;
def Tosa_RESIZE_BILINEAR          : I32EnumAttrCase<"BILINEAR", 2>;
```